### PR TITLE
HP-1473: new texts and styles

### PR DESCRIPTION
**Pull request by NikoHelle**
Updates to texts and text layouts.

This PR does not include major UI changes like "unboxing" Download and Delete Profile components or padding and colour changes. Those are in the Figma-layout linked in the Jira-ticket, but will be done in another PR.

### CODE REVIEW

File: src/i18n/en.json [hunk 1]
```diff
@@ -167,7 +172,7 @@
     "ariaSelectedOption": "{{value}} is selected",
     "ariaNoSelectedItemForLabel": "{{label}} is not selected",
     "deleteProfile": "Delete your profile",
-    "description": "The Helsinki profile enables you to use your data in all City of Helsinki online services that require authentication. One set of data, easily accessible on various services.",
+    "description": "The data stored in the Helsinki profile is used in the City of Helsinki’s e-services. You can find more detailed information <linkToServices>{{linkToServicesText}} on the tab</linkToServices>.",
```
Comment (akikoskinen) [inline src/i18n/en.json]: The end becomes "You can find more detailed information Your services on the tab". Should probably be changed to something better.

File: src/i18n/sv.json [hunk 2]
```diff
@@ -167,22 +172,21 @@
     "ariaSelectedOption": "{{value}} har valts",
     "ariaNoSelectedItemForLabel": "{{label}} har inte valts",
     "deleteProfile": "Radera profil",
-    "description": "Helsingfors-profilen gör det möjligt att använda dina uppgifter i alla Helsingfors webbtjänster som kräver identifiering. Samma information, enkel att använda i många olika tjänster.",
+    "description": "Uppgifterna som sparats i Helsingforsprofilen används i Helsingfors stads e-tjänster. Närmare information hittar du <linkToServices>{{linkToServicesText}} på fliken</linkToServices>.",
```
Comment (akikoskinen) [inline src/i18n/sv.json]: Samma problem här. Texten blir "Närmare information hittar du Dina tjänster på fliken".

File: src/i18n/fi.json [hunk 3]
```diff
@@ -33,9 +33,9 @@
...
-    "accept": "Ymmärrän mitä tietojeni poistaminen tarkoittaa ja haluan jatkaa.",
+    "accept": "Ymmärrän, mitä tietojeni poistaminen tarkoittaa ja haluan jatkaa.",
     "delete": "Poista omat tiedot",
-    "explanation": "Jos haluat, voit poistaa kaikki sinusta tallentamamme tiedot. Tietoja ei voi palauttaa sen jälkeen, kun ne on poistettu. Jos olet yhdistänyt profiilisi johonkin Helsingin kaupungin palveluun, et enää pysty käyttämään kyseistä palvelua ennen kuin luot uuden profiilin. Poistamisen jälkeen sinut kirjataan automaattisesti ulos tästä palvelusta. Jos olet käyttänyt kirjautumisessa ulkopuolisen palveluntarjoajan tiliä (esim. Google tai Facebook), kyseistä tiliä ei poisteta. ",
+    "explanation": "Halutessasi voit poistaa Helsinki-profiilisi ja kaikki sinusta tallentamamme tiedot. Tietoja ei voi palauttaa sen jälkeen, kun ne on poistettu. Helsinki-profiiliisi yhdistettyjä asiointipalveluja käyttääksesi on luotava uusi Helsinki-profiili.<br/><br/>Poistamisen jälkeen sinut kirjataan automaattisesti ulos tästä palvelusta.<br/><br/>Jos olet käyttänyt kirjautumisessa ulkopuolisen palveluntarjoajan tiliä (esim. Google tai Facebook), kyseistä tiliä ei poisteta.",
```
Comment (akikoskinen) [inline src/i18n/fi.json:38]: Would it be more correct to enclose paragraphs within `p` elements?

File: src/i18n/fi.json [hunk 4]
```diff
@@ -159,6 +160,10 @@
   },
   "profileInformation": {
     "address": "Osoite",
+    "addressDescription": "Nämä tiedot olet itse lisännyt Helsinki-profiili -palveluun tai muuhun Helsingin kaupungin asiointipalveluun. Tätä osoitetietoa käytämme silloin, kun olet tunnistautunut kevyesti, eli jotain muuta kuin Suomi.fi -tunnistautumista hyödyntäen. Voit halutessasi poistaa tiedot Helsinki-profiilista, jolloin ne poistuvat myös muista asiointipalveluista. Älä poista tietoja, jos sinulla on asiointi kesken.",
+    "addressDescriptionNoWeakAddress": "Et ole lisännyt toista osoitetta. Toista osoitetietoa käytämme silloin, kun olet tunnistautunut kevyesti, eli jotain muuta kuin Suomi.fi -tunnistautumista hyödyntäen.",
+    "addressDescriptionNoAddress": "Et ole lisännyt osoitetta. Tätä osoitetietoa käytämme silloin, kun olet tunnistautunut kevyesti, eli jotain muuta kuin Suomi.fi -tunnistautumista hyödyntäen.",
```
Comment (akikoskinen) [inline src/i18n/fi.json:165]: IMO these explanations are somewhat misleading. What address a service uses is not dependent on the user's (current) authentication method. Basically services can use the addresses in whatever way they want, only depending on what addresses a profile contains.

**Review by akikoskinen — CHANGES_REQUESTED** [review]
There are problems with the commit contents and/or order. Translation texts are removed before their usage is removed. Styles are used before they are introduced.

File: src/i18n/en.json (hunk 1 above)
Comment (NikoHelle) [reply src/i18n/en.json]: Changed!

File: src/i18n/fi.json (hunk 4 above)
Comment (NikoHelle) [reply src/i18n/fi.json:165]: True, but it was discussed that there is no need for deeper explanation.

File: src/i18n/fi.json (hunk 3 above)
Comment (NikoHelle) [reply src/i18n/fi.json:38]: The text is inserted in to a &lt;p&gt; element in the code. &lt;br&gt; is better in this case between text blocks.

**Review by akikoskinen — APPROVED** [review]

**Pull request merged**